week-3-practise/level-2/2-map.js

```diff
@@ -13,8 +13,11 @@ var attendanceCounts = [
   ["Tayoa", 11],
   ["Nina", 10]
 ];
+function nameOfStudent (name,index) {
+  return 
+}
 
-var eligibleStudentNames; // TODO: Complete this line.
+var eligibleStudentNames = ; // TODO: Complete this line.
```

Comment:
Looks like this isn't finished?

Reply:
It is finished on my machine, I just checked it, I don't know why it is now showing there